### DISCUSSION TITLE
Robotframework test execution from a Task

### DISCUSSION THREAD

**Discussion by 95rade** [tektoncd/pipeline#5100] [General]
Has anyone successfully executed robotframework tests from a Task or a Pipeline? I tried a Task from Tekton HUB but it fails with:

'Defaulted container "step-robot-tests" out of: step-robot-tests, place-scripts (init), working-dir-initializer (init), place-tools (init)'

Anyone?

**Comment by AlanGreene** [comment]
This was already opened as an issue and is being discussed there, see https://github.com/tektoncd/pipeline/issues/5092
The 'Defaulted container' message is not an error

**Reply by 95rade** [reply]
Thanks Alan, I am troubleshooting the task in question because it appears to be a flaky code and should have not been admitted to tekton HUB without prior testing.
